packages/trader/src/AppV2/Components/TradeParameters/RiskManagement/take-profit-and-stop-loss-input.tsx

```diff
@@ -168,7 +176,8 @@ const TakeProfitAndStopLossInput = ({
 
         /* In order to get validation params for Multipliers when TP and SL are empty, 
             we send '1' first, get validation params and set them into the state.*/
-        const input_value = should_set_validation_params ? '1' : new_input_value;
+        const default_value = is_crypto_currency ? '0.00002' : '1';
```

Comment:
Is `0.00002` sufficient for all the cryptocurrencies? There are different cryptocurrencies with different decimal points, including BTC, ETH or USDT.

Reply:
Agree, seem to be stupid mistake. BE card with fixes is not ready yet, so there is no way I can get acceptable range for crypto. Let me ask PO Kate if it's okay not to show Acceptable range info for crypro multipliers from the beginning (temporary, while we are waiting for BE)

Reply:
Conformed with Kate, she doesn't mind. We can proceed like this (not showing Acceptable Range for crypto Multiplier from the very begging). When BE will be ready (card is in progress), I refactor this logic